cats/cats-redis/src/main/java/com/netflix/spinnaker/cats/redis/cluster/ClusteredAgentScheduler.java

```diff
@@ -102,7 +104,7 @@ public ClusteredAgentScheduler(RedisClientDelegate redisClientDelegate,
     Set<String> skip = new HashSet<>(activeAgents.keySet());
     Integer availableAgents = maxConcurrentAgents - skip.size();
     if (availableAgents <= 0) {
-      logger.debug("Not acquiring more locks (activeAgents: " + skip.size() + " >= maxConcurrentAgents " + maxConcurrentAgents + ")");
+      logger.debug("Not acquiring more locks (activeAgents: " + skip.size() + " >= " + maxConcurrentAgents + "). Running agents: " + skip.stream().sorted().collect(Collectors.joining(",")));
```

Comment:
lets get this merged and pushed ...

as an aside, try and avoid lines < 120 characters ... 

```
logger.debug(
  "Not acquiring more locks (activeAgents: {}, runningAgents: {})",
  skip.size(),
  skip.stream().sorted().collect(Collectors.joining(","))
)
```

^^ makes it easy to do for logs

Reply:
Sounds good to all. Will make those changes!